src/query/algorithms/triangles/StreamingTriangles.cpp

```diff
@@ -206,6 +207,7 @@ std::string StreamingTriangles::countDynamicCentralTriangles(
         std::vector<std::string> oldCentralRelationCount) {
     streaming_triangle_logger.info("###STREAMING TRIANGLE### Dynamic Streaming Central Triangle "
                                   "Counting: Started");
+    std::string joinedString = partitionIdList[0] + partitionIdList[1] + partitionIdList[2];
```

Comment:
Are we sure that always the `partitionIdList` size is not less than 3?

Reply:
I have solved this

Reply:
Ok